src/sentry/runner/commands/devserver.py

```diff
@@ -201,6 +201,8 @@ def devserver(
     needs_https = parsed_url.scheme == "https" and (parsed_url.port or 443) > 1024
     has_https = shutil.which("https") is not None
 
+    control_silo_port = port + 10
```

Comment:
should this use the same env var from above if it's set? `SENTRY_CONTROL_SILO_PORT`